pages/web-literacy/color.js

```diff
@@ -0,0 +1,33 @@
+var categories = require('./categories');
```

Comment:
Right now this file just seems to be storing key names to English words, so it makes it hard to type "Problem-Solving" in one spot and "problem solving" in another. Keeps it consistent. We jut interact with keys.

I don't mind keeping it, but it's really not required.